doc/6-distributed-monitoring.md

```diff
@@ -636,7 +636,7 @@ The endpoint configuration could look like this, for example:
       host = "192.168.56.101"
     }
 
-    object Endpoint "icinga2-client2.localdomain" {
+    object Endpoint "icinga2-client1.localdomain" {
       host = "192.168.56.112"
```

Comment:
client1 uses 56.111, client2 uses 56.112 for better readability. Currently top down config sync uses an example client called `client1` whereas the top down command endpoint chapters use `client2`. Due to the changed order this might be confusing.